### ISSUE TITLE
Sharpness addon: make "Effect strength" live

### ISSUE THREAD

**Issue by humdingerb** [HaikuArchives/ArtPaint#92]
While the "Sharpness" slider takes effect while you move the slider's knob, the "Effect strength" slider only takes effect after letting go of the knob. All settings should be live, if possible.

**Comment by dsizzle** [comment]
This might not be possible given the way the Sharpness add-on works.  It looks like there's a blur map that gets generated based on the effect size, and it takes noticeable time.  It can't update that map fast enough to give a good 'live' experience with this current implementation.

**Comment by dsizzle** [comment]
Yes, let's leave it open.  There could be an answer in Wonderbrush, or maybe right here in ArtPaint.  The Blur addon works live so maybe it does something smarter.

**Comment by humdingerb** [comment]
Thanks for trying.
I'm not that familiar with threading and stuff, but naively I'd have assumed that in the computation loop, the thread could check some flag that changes when the slider value changes. If it does, exit the thread immediately and start a new thread with the new value. Or maybe restart the calculation with the new value directly?
I dunno, as I said, my naivety and not being a good coder... :)
